server/api/users.js

```diff
@@ -1,17 +1,61 @@
-const router = require('express').Router()
-const { models: { User }} = require('../db')
-module.exports = router
+const router = require("express").Router();
+const {
+  models: { User, Posture },
+} = require("../db");
+module.exports = router;
 
-router.get('/', async (req, res, next) => {
+router.get("/", async (req, res, next) => {
   try {
     const users = await User.findAll({
       // explicitly select only the id and username fields - even though
       // users' passwords are encrypted, it won't help if we just
       // send everything to anyone who asks!
-      attributes: ['id', 'username']
-    })
-    res.json(users)
+      attributes: ["id", "username"],
+    });
+    res.json(users);
   } catch (err) {
-    next(err)
+    next(err);
   }
-})
+});
+
+//GET /users/:id/poses (get all)
+router.get("/:id/poses", async (req, res, next) => {
+  try {
+    //token check, in axios, set authorization header
+    const user = await User.findByToken(req.headers.authorization);
+    const poses = await Posture.findAll({ where: { userId: user.id } });
+    res.send(poses);
+  } catch (error) {
+    next(error);
+  }
+});
+
+//POST /users/:id/pose (add pose)
+router.post("/:id/pose", async (req, res, next) => {
+  try {
+    const user = await User.findByToken(req.headers.authorization);
```

Comment:
You should probably also cross-check the user's id against the route parameter. The route silently creates a resource for the user from the token (which is safe because they aren't able to post to someone else's account), but it would probably be more correct to throw an error